linux/net/rina/common.c

```diff
@@ -39,7 +39,7 @@ EXPORT_SYMBOL(is_port_id_ok);
 #define CEP_ID_WRONG -1
 
 bool is_cep_id_ok(cep_id_t id)
-{ return true; /* FIXME: Bummer, add it */ }
+{ return id >= 0 ? true : false; }
```

Comment:
can you explain the logic under this modification? just a safety check or impacts some places where the function is used?
(btw, it is more compact to use "return id >=0")